### ISSUE TITLE
Module '"supertokens-auth-react/ui"' has no exported member 'getSuperTokensRoutesForReactRouterDom'

### ISSUE THREAD

**Issue by Yanis02015** [supertokens/supertokens-core#664]
## 🐛 Bug Report

With a development stack: Vitejs, React, I get this Typescript error when I import getSuperTokensRoutesForReactRouterDom

> Module '"supertokens-auth-react/ui"' has no exported member 'getSuperTokensRoutesForReactRouterDom'. Did you mean to use 'import getSuperTokensRoutesForReactRouterDom from "supertokens-auth-react/ui"' instead?

My code is available here
[Import_getSuperTokensRoutesForReactRouterDom](https://github.com/Yanis02015/import_getSuperTokensRoutesForReactRouterDom.git)

It's just a TypeScript error, so on my browser everything works perfectly. So currently I use the @ts-ignore

## Useful information

I get the same error if I use the tsc command

![Capture d’écran 2023-05-03 à 21 10 50](https://user-images.githubusercontent.com/8948640/236244129-ce3fb9ae-81b8-4a40-bbdb-e2666d94ba09.png)

![Capture d’écran 2023-05-04 à 15 49 07](https://user-images.githubusercontent.com/8948640/236244586-ed4889a8-f5e4-4b3c-bb33-799ba2e66c2f.png)

**Comment by Sohett** [comment]
+1

**Comment by Sohett** [comment]
It seems like the correct import is the following:

```
import getSuperTokensRoutesForReactRouterDom from 'supertokens-auth-react/ui';
```

But that's clearly not what currently in the [doc](https://supertokens.com/docs/passwordless/pre-built-ui/setup/frontend)

**Comment by rishabhpoddar** [comment]
`import {getSuperTokensRoutesForReactRouterDom} from 'supertokens-auth-react/ui';` should also work - we will be updating the SDK to fix this issue today

**Comment by Sohett** [comment]
> It seems like the correct import is the following:
> 
> ```
> import getSuperTokensRoutesForReactRouterDom from 'supertokens-auth-react/ui';
> ```
> 
> But that's clearly not what currently in the [doc](https://supertokens.com/docs/passwordless/pre-built-ui/setup/frontend#3-setup-routing-to-show-the-login-ui)

Error from my side. It's not working. 

It's just not throwing any errors so it looks like it's working. The correct implementation is this one:

```
import {getSuperTokensRoutesForReactRouterDom} from 'supertokens-auth-react/ui';
```

Problem is that the typescript error makes it fail on build. App can't be deployed because of it

**Comment by rishabhpoddar** [comment]
We just released a new version of the SDK -> 0.32.2 which should have fixed this issue.

The right way of importing is 

```
import {getSuperTokensRoutesForReactRouterDom} from 'supertokens-auth-react/ui';
```

I'm closing this issue for now, but feel free to reopen if required.
